.github/workflows/api_refs.yaml

```
@@ -15,6 +15,7 @@ jobs:
             with:
                 repository: ibexa/documentation-developer
                 workflow: 'api_refs.yaml'
+                ref: ${{ github.ref_name }}
```

The right workflow is on default branch, that's why I didn't add this parameter in the first place.
```suggestion
                ref: 5.0
```